### PR TITLE
Fix readme for local env

### PR DESCRIPTION
**Pull request by amoghjalan**
## Acceptance Criteria fulfillment

- [ ] Enables other users to replicate the README.md instructions and let's them start local server

### CODE REVIEW

File: database-docker/docker-compose.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ services:
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
-      POSTGRES_DB: dora
+      POSTGRES_DB: mhq-oss
```
Comment (samad-yar-khan) [inline database-docker/docker-compose.yml:10]:
```suggestion
      POSTGRES_DB: middleware
```

File: database-docker/docker-compose.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ services:
     volumes:
       - ./db:/db
     environment:
-      - DATABASE_URL=postgres://postgres:postgres@db:5432/dora?sslmode=disable
+      - DATABASE_URL=postgres://postgres:postgres@db:5432/mhq-oss?sslmode=disable
```
Comment (samad-yar-khan) [inline database-docker/docker-compose.yml:26]:
```suggestion
      - DATABASE_URL=postgres://postgres:postgres@db:5432/middleware?sslmode=disable
```

**Review by samad-yar-khan — CHANGES_REQUESTED** [review]

File: database-docker/docker-compose.yml (hunk 2 above)
Comment (amoghjalan) [reply database-docker/docker-compose.yml:26]: The whole application uses `mhq-oss` in all environments. This shall mean change over their as well

File: database-docker/docker-compose.yml (hunk 1 above)
Comment (amoghjalan) [reply database-docker/docker-compose.yml:10]: The whole application uses `mhq-oss` in all environments. This shall mean change over their as well

File: database-docker/docker-compose.yml (hunk 2 above)
Comment (samad-yar-khan) [reply database-docker/docker-compose.yml:26]: ahh pain

**Review by samad-yar-khan — APPROVED** [review]

**Pull request merged**